tfexec/cmd.go

```diff
@@ -275,3 +301,105 @@ func writeOutput(ctx context.Context, r io.ReadCloser, w io.Writer) error {
 		}
 	}
 }
+
+func (tf *Terraform) runTerraformCmdWithGracefulShutdown(ctx context.Context, cmd *exec.Cmd) error {
+	return tf.runTerraformCmdInner(ctx, cmd, true /* enableGracefulShutdown */)
+}
+
+func (tf *Terraform) runTerraformCmd(ctx context.Context, cmd *exec.Cmd) error {
+	return tf.runTerraformCmdInner(ctx, cmd, false /* enableGracefulShutdown */)
+}
+
+func (tf *Terraform) runTerraformCmdInner(ctx context.Context, cmd *exec.Cmd, enableGracefulShutdown bool) error {
+	var errBuf strings.Builder
+
+	cmd.SysProcAttr = defaultSysProcAttr
```

Comment:
Use different value for `cmd.SysProcAttr` due to the difference values for different platforms.